<!DOCTYPE html>
<html>
<head>
  <title> Tracey Oellerich </title>
  <meta name="keywords" content="">
    <meta name="description" content="">
    <meta name="HandheldFriendly" content="true">
    <meta name="MobileOptimized" content="320">
    <script>
      if(navigator.userAgent.match(/hpwOS/i) || navigator.userAgent.match(/webOS/i)){
        document.write("<meta name=\"viewport\" content=\"width=device-width, target-densitydpi=160, minimum-scale=1.0\">");
      } else {
        document.write("<meta name=\"viewport\" content=\"width=device-width, target-densitydpi=160, minimum-scale=1.0, maximum-scale=1.0, user-scalable=no\">");
      }
    </script><meta name="viewport" content="width=device-width, target-densitydpi=160, minimum-scale=1.0, maximum-scale=1.0, user-scalable=no">
 <meta name="viewport" content="width=device-width, initial-scale=1"/>
<style>
* {box-sizing: border-box}

/* Set height of body and the document to 100% */
body, html {
  height: 100%;
  margin: 0;
  font-family: Times New Roman;
}

/* Style tab links */
.tablink {
  background-color: #303030;
  color: white;
  float: left;
  border: none;
  outline: none;
  cursor: pointer;
  padding: 14px 16px;
  font-size: 19px;
  font-family: Times New Roman;
  width: 16.65%;
}

.tablink:hover {
  background-color: #777;
}

/* Style the tab content (and add height:100% for full page content) */
.tabcontent {
  color: black;
	font-size: 18px;
  display: none;
  padding: 100px 20px;
  height: 100%;
}

#Home {background-color: white;}
#Research {background-color: white;}
#Teaching {background-color: white;}
#Outreach {background-color: white;}
#Links{background-color: Links;}
</style>
</head>
<body>

<button class="tablink" onclick="openPage('Home', this, 'black')"id="defaultOpen">Home</button>
<button class="tablink" onclick="openPage('Research', this, 'black')" >Research</button>
<button class="tablink" onclick="openPage('Teaching', this, 'black')">Teaching</button>
<button class="tablink" onclick="openPage('Outreach', this, 'black')">Leadership and Outreach</button>
<button class="tablink" onclick="openPage('Awards', this, 'black')">Honors and Awards</button>
<!-- <button class="tablink" onclick="openPage('CV', this, 'black')">Curriculum Vitae</button> -->
<button class="tablink" onclick="openPage('Links', this, 'black')">Links</button>

<div id="Home" class="tabcontent">
	<h1> Tracey Oellerich </h1>
	<meta name="viewport" content="width=device-width, initial-scale=1">
	<style>
	* {
	  box-sizing: border-box;
	}

	.row {
	  display: flex;
	}


	.column {
  float: left;
  padding: 10px;
  height: 300px; /* Should be removed. Only for demonstration */
}

.left {
  width: 25%;
}

.right {
  width: 75%;
}
	</style>
	</head>
	<body>
		<div class="row">
		<div class="column" style="background-color:white;">
				<img src="tracey.jpg" width="330" height="560" align="middle">

		</div>
	  <div class="column" style="background-color:white;">
			<br>
			<br>
			<br>
			<font size="+1.6"> I am a final year PhD student in the Department of Mathematics at George Mason University working under the supervision of Prof. <a href="https://math.gmu.edu/~memelian/">Maria Emelianenko</a>. I have a Masters degree in Mathematics from George Mason University.
			   I completed my BS in Mathematics and BA in Physics with minors in Statistics and Secondary Education from Wilkes University, Wilkes-Barre, PA in 2016.
			 <br>
			 <br>
				 I am working on interdisciplinary research at the intersection of Mathematics and Biology. I am primarily interested in data science and network science, which can be used to model biological structures and analyze them. In particular, I work with biological networks which are systems formed by the interactions of a finite number of molecules such as proteins, RNA transcripts, genes, or various combinations. This work is helpful in identifying effective drug treatments for adapting diseases, such as cancer, in which a drug (external input) being applied to the system is being neutralized within the network. My work incorporates both theoretical and experimental mathematics in order to identify these features. Utilizing aspects of linear algebra and graph theory, I work to understand how and why a biological network will adapt to external inputs. Then, using experimental data, I employ novel data science and machine learning techniques to infer dynamics and create a dynamical system model.
				 During my undergraduate years at Wilkes University, I was introduced to the idea of utilizing mathematics in biology applications. I began in the field of computational biophysics where I worked on “Enhanced protein folding through confinement inside a hydrophilic nanopore” under the supervision of Prof. <a href="https://www.wilkes.edu/campus-directory/del.lucent.aspx">Del Lucent</a>. For this work, we investigated the complexity of protein folding inside a cell, in particular, under confinement. Proteins are large molecules composed of one or more long chains of amino acids. Protein folding is the process in which a protein reaches its functional shape. In this heterogeneous environment, proteins can fold inside of special nanocages called chaperones, which are cone-shaped proteins that assist other proteins in folding or unfolding. Through my work, I have developed a model describing how these cages can accelerate protein folding by modifying the thermodynamics of confined water. I tested these models through distributed computing and molecular dynamics simulation and used state of the art master equation appr				oaches to analyze the results and verify my model.
				During a study abroad in Padua, Italy, I had the opportunity to participate in the “Bench to Bedside: Translational Molecular Research” course. This course focused on the practical use of genomics, proteomics and bioinformatics for the diagnosis, prevention and treatment of disease. It was co-taught by faculty from US and Italy including experts in the field of medical research on topics such as leukemia, cancer research, infectious diseases, etc. The course utilized hands-on technical workshops and touched on issues related to implementation of new platforms and assays. Through both presentations and experiments, I was given the opportunity to see where the data I work with is collected, the challenges involved in this process, and a different prospective on how it is used in research.
				In the summer of 2021, I was given the opportunity to participate as a Data Science intern at the National Center for Advancing Translational Sciences (NCATS) at the National Institutes of Health (NIH). There, I worked on using deep learning to prioritize disease targets for protein-protein interaction networks. We worked on utilizing the data in the Pharos library of known diseases and protein targets to create a ranked list of protein targets by modifying the GuiltyTargets algorithm, a machine learning algorithm that ranks protein targets using gene expression and positive-unlabeled machine learning. This work helps to shave years off the drug development process by eliminating unsuitable targets without the need for testing.

				<br>
				<br>
<br>
<br>
<p>
    <h1>Curriculum Vitae</h1>
    <p>Curriculum Vita: <a href="Oellerich_cv.pdf">[pdf]</a>.</p>
  </body>
</html>
  </p>
<br>
<br>
				<h3>Contact</h3>
		<p>
		</p><ul>
			<li>E-mail: <a href="mailto:toelleri@gmu.edu">toelleri@gmu.edu</a>.</li>
			<li> Mail: <span class="mail">
						Department of Mathematical Sciences, <br>
						George Mason University<br>
						4400 University Drive, MS 3F2 <br>
			Fairfax, Virginia - 22030
						 </li>
			<li>Office:  Exploratory Hall, Room 4310.</li>
		</ul>
		 </font>
	</div>
	</div>


</div>

<div id="Research" class="tabcontent">
	
  <h3>Research Interests </h3>
    <ul>
      <li> Mathematical Biology </li>
      <li> Data Science </li>
      <li> Network Science </li>
      <li> Numerical Methods </li>
      <li> Dynamical Systems </li>
      <li> Machine Learning </li>
      <li> Modeling of Nonlinear Systems </li>
      <li> Optimization </li>
    </ul> 
<br>

 <h3> Research </h3>
	<ul>
		<li> George Mason University (GMU) </li>
		<ul>
			<li> 2018 - present: <i>Network Analysis of Biological Systems: Adaptation and Inferring Dynamics</i> 
			Advisor: Dr. M. Emelianenko, Department of Mathematical Sciences </li>
		</ul>
		<li> National Institutes of Health (NIH) </li>
		<ul>
			<li> Summer 2021: <i>Deep Learning on Embedded Protein-Protein Interaction Networks to Prioritize Disease Targets</i>
				Mentor: Dr. Vishal Siramshetty, National Center for Advancing Translational Sciences (NCATS) </li>
		</ul>
		<li> Wilkes University </li>
		<ul>
			<li> 2015 - 2016: <i>Enhanced Protein Folding through Confinement Inside a Hydrophilic Nanopore</i>
				Advisor: Dr. D. Lucent, Department of Physics	</li>
			<li> Spring 2015: <i>Squaring the Circle using Hyperbolic Geometry</i>
				Advisor: Dr. L. Berard, Department of Mathematics	</li>
			<li> Fall 2014: <i>Stern’s Diatomic Sequence</i> with E. Klemchak 	
				Advisor: Dr. R. Pryor, Department of Mathematics </li>
		</ul>
	</ul>
<br>




	
<h3> Publications</h3>


          <ol>
		<li> T. Oellerich, M. Emelianenko, M. Pierobon, and E. Baldelli. <i>Learning biological network dynamics from data.</i> (In Preparation). </li>
		<li> T. Oellerich, M. Emelianenko, L. A. Liotta, and R. P. Araujo. <i>Biological networks with singular jacobians: their origins and adaptation criteria.</i> Submitted. doi: <a href="https://doi.org/10.1101/2021.03.01.433197">https://doi.org/10.1101/2021.03.01.433197</a>.
          </ol>
<h3> Selected Presentations and Talks </h3>
<ul>
<!-- <li> This is a comment</i></li> -->
	<li> July 11, 2022: <i>Inferring Dynamics of Biological Systems</i>, SIAM Conference on the Life Sciences, Pittsburg, PA</li>
	<li> May 18, 2022: <i>Inferring Dynamics of Biological Systems</i>, Biology and Medicine through Mathematics (BAMM!), Richmond, VA</li>
	<li> April 8, 2022: <i>Network Analysis of Biological Systems: Adaptation and Inferring Dynamics</i>, 3 Minute Thesis (3MT) Competition, George Mason University, Fairfax, VA</li>
	<li> April 6, 2022: <i>Singular Jacobians and Their Effect on Adaptation in Biological Networks</i>, Joint Mathematics Meeting, Online</li>
	<li> January 28, 2022: <i>Network Analysis of Biological Systems: Adaptation and Inferring Dynamics</i>, Industrial Immersion Program, Student Research Talks, George Mason University, Fairfax, VA</li>
	<li> September 24, 2021: <i>Singular Jacobians and Their Effect on Adaptation in Biological Networks</i>, We Speak: Early-Career Mathematicians Lightning Talks, Association for Women in Mathematics, Online</li>
	<li> January 18, 2020: <i>Adaptability Conditions in Biological Networks</i>, Joint Mathematics Meeting, Denver, CO</li>
	<li> September 13, 2019: <i>Adaptability Conditions in Biological Networks</i>, Student Research Talks, George Mason University, Fairfax, VA</li>
	<li> March 1, 2019: <i>An Introduction to Robust Perfect Adaptation Networks</i>, Student Research Talks, George Mason University, Fairfax, VA</li>
	<li> October 4, 2017: <i>Dirac Measure</i>, PDE Seminar, George Mason University, Fairfax, VA  </li>
	<li> May 26, 2017: <i>Numerical Solutions to Nonlinear Equations</i>, George Mason University, Fairfax, VA</li>
</ul>
<h3> Selected Poster Presentations </h3>
<ul>
	<li> July 12, 2022: <i>Inferring Dynamics of Biological Systems</i>, AWM Poster Session at the SIAM Annual Meeting, Pittsburgh, PA  </li>
	<li> April 8, 2022: <i>Inferring Dynamics of Biological Systems</i>, AWM Poster Session at the Joint Mathematics Meeting, Online  </li>
	<li> August 5, 2021: <i>Deep Learning on Embedded Protein-Protein Interaction Networks to Prioritize Disease Targets</i>, National Institutes of Health Summer Research Presentations, Bethesda, MD  </li>
	<li> July 10, 2020: <i>Mathematical Conditions for Adaptation in Biological Networks</i>, AWM Poster Session at the SIAM Annual Meeting, Online</li>
	<li> February 17, 2020: <i>Mathematical Conditions for Adaptation in Biological Networks</i>, Southeast Center for Mathematical Biology Symposium,  Atlanta, Georgia </li>
	<li> January 28, 2019: <i>Exploring Robust Perfect Adaptation</i>, Southeast Center for Mathematical Biology Symposium,  Atlanta, Georgia </li>
</ul>
	<br>
	<br>
	<br>
	<br>
	<br>


</div>

<div id="Teaching" class="tabcontent">
	
<h3>Teaching Experience</h3>
<ul>
	<li> Instructor of Record </li>
	<br>
	<ul>
		<li> Summer 2020 - MATH 446/OR 481: Numerical Analysis I </li>
		<li> Summer 2020 - MATH 108: Introductory Calculus with Business Applications </li>
		<li> Summer 2019 - MATH 446/OR 481: Numerical Analysis I </li>
		<li> Summer 2018 - MATH 111: Linear Math Modeling </li>

	<br>
	</ul>
		
	<li> Graduate Teaching Assistant </li>
	<br>
	<ul>
		<li> Spring 2018  -  MATH 213: Analytic Geometry/Calculus III</li>
		<li> Fall 2017  - MATH 114: Analytic Geometry and Calculus 2</li>
		<li> Spring 2017 - MATH 114: Analytic Geometry and Calculus 2</li>
		<li> Fall 2016  - MATH 113: Analytic Geometry and Calculus 1</li>
	
</ul>
		
  <p> </p>
</div>

<div id="Outreach" class="tabcontent">
  <h3><a href="https://streets-gmu.wikidot.com/"> Student Research Talks (StReeTs) at GMU</a></h3>
<ul>
    <li> Spring 2022 : Co-organizer </li>
    <li> Fall 2021 : Co-organizer </li>
    <li> Fall 2020 : Organizer </li>
    <li> Spring 2020 : Co-organizer </li>
    <li> Fall 2019   : Co-organizer</li>
    <li> Spring 2019 : Co-organizer</li>
    </ul>

  <h3> <a href="https://www.siam.org/">Society for Industrial and Applied Mathematics - GMU Chapter   </a>  </h3>
  <ul>
    <li> March 2021 - March 2022 : Treasurer </li>
    <li> March 2020 - March 2021 : Vice-President </li>
    <li> March 2018 - March 2020 : President </li>
    <li> March 2017 - March 2018 : Treasurer </li>
    <li> March 2018: Member of the SIAM Festivals Working group </li>
  </ul>
<br>
    <h3><a href="https://gapsa.gmu.edu/">Graduate and Professional Student Association (GAPSA) </a>, George Mason University</h3>
    <ul>
      <li> January 2020 - May 2021: Student Representative for the Mathematics Graduate Program </li>
    </ul>
<br>
    <h3> <a href="https://sites.google.com/site/gmuextreems/"> EXTREEMS-QED Undergraduate Research Program </a>, George Mason University</h3>
	<ul>
		<li> Summer 2018: Mentor </li>
	</ul>
<br>


    If you are interested in or need more information about any of the activities listed here, feel free to send me an email. My contact info is listed on the home page.
</div>

<div id="Awards" class="tabcontent">

  <h3>Honors </h3>
    <ul>
      <li> September 18-23, 2022: One of 200 students selected worldwide to participate in the 9th Heidelberg Laureate Forum, Heidelberg, Germany  </li>
      <li> June 5 – June 11, 2022: Participant in the American Mathematical Society’s Mathematics Research Community (MRC) on Models and Methods for Sparse (Hyper)Network Science, Java Center, NY  </li>
      <li> Summer 2022: Recipient of the 2022 Summer Research Fellowship through the George Mason University Provost, Fairfax, VA</li>
      <li> April 8, 2022: Finalist in George Mason University’s 3MT® (Three-Minute Thesis) Competition, Fairfax, VA</li>
      <li> September 19-24, 2021:  One of 200 students selected worldwide to participate in the 8th Heidelberg Laureate Forum, Online  </li>
      <li> Summer 2021:  One of 15 students selected to participate in the Graduate Data Science Summer Program (GDSSP) at the National Institutes of Health (NIH), Bethesda, MD</li>
      <li> October 3-4, 2019: Participant in Equity in Education Data-thon, Library of Virginia, Richmond, VA  </li>
      <li> May 6 - 10, 2019: Travel grant to participate in the Fields-CQAM Industrial Problem Solving Workshop (IPSW), Fields Institute, Toronto, Canada </li>
      <li> Fall 2018-Spring 2022: Awarded Research Funding through the George Mason Industrial Immersion Program, Fairfax, VA </li>
    </ul> 

<br>
  
  <h3>Awards</h3>
    <ul>
      <li> May 2017: Received the Teaching Award in Mathematics, George Mason University, Fairfax, VA </li>
      <li> May 2016: Received the James DeCosmo Award in Mathematics, Wilkes University, Wilkes-Barre, PA  </li>
      <li> May 2016: Received the Frederick E. Bellas Award for Outstanding Physics Student, Wilkes University, Wilkes-Barre, PA    </li>
      <li> May 2016: Received the College of Science and Engineering Outstanding Student Award, Wilkes University, Wilkes-Barre, PA  </li>
    </ul>


</div>



<!-- <div id="CV" class="tabcontent">

  <p>
    <h1>Curriculum Vitae</h1>
    <p>Download <a href="CV_Thomas,Cigole.pdf">CV</a>.</p>
  </body>
</html>
  </p>
</div>
-->

<div id="Links" class="tabcontent">
<!--
  <p>
    <h1>Some useful links.</h1>
	<br>
	Some links that I find interesting or helpful.
	<h3> National Mathematical Organizations</h3>
	<ul> 
		<li> <a href ="https://www.ams.org/home/page">American Mathematical Association(AMS)</a></li>
		<li> <a href ="https://www.maa.org">Mathematical Association of America(MAA)</a></li>
		<li> <a href ="https://www.siam.org">Society for Industrial and Applied Mathematics(SIAM)</a></li>
		<li> <a href ="https://awm-math.org">Association for Women in Mathematics(AWM)</a></li>
		</ul>
	<h3> Opportunities </h3>
	<ul>  
		<li> <a href ="https://www.mathprograms.org">MathPrograms:</a> To search and apply for internships </li>
		<li> <a href ="https://www.mathjobs.org">MathJobs</a>: To search and apply for jobs in mathematics(postdocs, teaching positions etc.)  </li>
		
		
	</ul>
	<h3> Some Interesting Articles </h3>
<ul>
	<li> A nice article on arithmetic dynamics: <a href ="https://www.quantamagazine.org/mathematicians-set-numbers-in-motion-to-unlock-their-secrets-20210222/"> Mathematicians Set Numbers in Motion to Unlock Their Secrets</a>, Quanta Magazine</li>
</ul>
  </body>
</html>
  </p>
-->
</div>

<script>
function openPage(pageName,elmnt,color) {
  var i, tabcontent, tablinks;
  tabcontent = document.getElementsByClassName("tabcontent");
  for (i = 0; i < tabcontent.length; i++) {
    tabcontent[i].style.display = "none";
  }
  tablinks = document.getElementsByClassName("tablink");
  for (i = 0; i < tablinks.length; i++) {
    tablinks[i].style.backgroundColor = "";
  }
  document.getElementById(pageName).style.display = "block";
  elmnt.style.backgroundColor = color;
}

// Get the element with id="defaultOpen" and click on it
document.getElementById("defaultOpen").click();
</script>

</body>
</html>
